lib/SetAPI/generic/SetInterfaceV1.py

```diff
@@ -44,8 +47,20 @@ def save_set(
         :rtype: dict[str, Any]
         """
         self._check_save_set_params(params)
-        save_params = self._build_ws_save_obj_params(set_type, provenance, params)
-        return self.ws.save_objects(save_params)
+        save_params = {
+            "objects": [
+                {
+                    "name": params["output_object_name"],
+                    "data": params["data"],
+                    "type": set_type,
+                    "provenance": provenance,
+                    "hidden": 0,
+                }
+            ]
+        }
+        ws_params = convert_workspace_param(params)
```

Comment:
this was in a separate function but may as well just spell it out here as it isn't particularly long or complex